index.bs

```diff
@@ -2890,35 +5011,134 @@ partial interface MLGraphBuilder {
                   builder.constant(1))),
               builder.constant(options.steepness));
     </pre>
-    </div>
+  </details>
+</div>
+
+{{MLSoftplusOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLSoftplusOptions>
+    : <dfn>steepness</dfn>
+    ::
+         A {{float}} scalar parameter.
+         The default value is `1`.
+</dl>
+
+#### The {{MLGraphBuilder/softplus(input, options)}} method #### {#api-mlgraphbuilder-softplus-input-options}
+<div>
+    **Arguments:**
+        - *input*: an {{MLOperand}}. The input tensor.
+        - *options*: an optional {{MLSoftplusOptions}}. The optional parameters of the operation.
+
+    **Returns:**
+        - an {{MLOperand}}. The output tensor of the same shape as *input*.
 </div>
 
-### The softsign() method ### {#api-mlgraphbuilder-softsign}
+<details open>
+  <summary>
+    The {{MLGraphBuilder/softplus(input, options)}} method steps are:
+  </summary>
+  <div algorithm=softplus-input-options class=algorithm-steps>
+    1. If any of the following sub-steps fail, [=exception/throw=] an "{{OperationError}}" {{DOMException}}.
```

Comment:
Consider adding a check for input
```
 1. [=Assert=]: the type of |input| is {{MLOperand}}.
```

Reply:
not needed, WebIDL bindings take care of that